src/utils/appLinking.js

```diff
@@ -0,0 +1,147 @@
+import { Linking } from "react-native";
+import { Alert } from "../components";
+import {
+	team,
+	general,
+	announcements,
+	fundraisingCommittee,
+	motorshpe,
+	projects20_21,
+	shpejr,
+	mentorshpe20_21,
+	computereng,
+	projectleads,
+	internationalStudents,
+	linkedinDrop
+} from "react-native-dotenv";
+/**
+ * @description Object that holds the slack team id and several channel ids.
+ *              The information in here is the most current as of May 28th, 2020.
+ *              To find the team id or channel id login in slack on a computer.
+ *
+ *  slackInfo
+ */
+
+export const slackInfo = {
+	general: general,
+	announcements: announcements,
+	fundraisingCommittee: fundraisingCommittee,
+	motorshpe: motorshpe,
+	projects20_21: projects20_21,
+	shpejr: shpejr,
+	mentorshpe20_21: mentorshpe20_21,
+	computereng: computereng,
+	projectleads: projectleads,
+	internationalStudents: internationalStudents,
+	linkedinDrop: linkedinDrop
+};
+
+/**
+ * @description Function that returns the URI scheme and universal link for opening
+ *              an app, along with parameters, if included. Should not be changed
+ *              unless the apps change the universal links and/or URI schemas.
+ *
+ * @param {String}  inquiry  The value of the query.
+ * @param {String=} intent   The value that specifies the type of query.
+ *
+ * @example
+ * // How to call the function
+ * formatUrl[name_of_the_app](inquiry, intent);
```

Comment:
I'd rather use camelCase as `nameOfTheApp`, `nameOfApp`, or `appName` rather than snake_case

Reply:
took care of it